### DISCUSSION TITLE
Code plugin includes merges of other peoples code

### DISCUSSION THREAD

**Discussion by jayvdb** [lowlighter/metrics#639] [Ideas]
The [code plugin](https://github.com/lowlighter/metrics/tree/master/source/plugins/code) includes snippets authored by others which are a merge commit.

e.g. my [current snippet](https://github.com/jayvdb/jayvdb/commit/9dabff00eac6842f0f1a0a3febc97d44aec6d6d2) is from https://github.com/jayvdb/pypidb/commit/2af9d402b81da6aa5598ea6e1d5015b48a75d20d , which is a bog standard merge of https://github.com/jayvdb/pypidb/pull/120

**Comment by lowlighter** [comment]
(Tracked in #646)

**Reply by lowlighter** [reply]
Should be fixed now
